test/src/test/java/org/corfudb/integration/CorfuStoreBrowserEditorIT.java

```diff
@@ -869,9 +883,13 @@ public void addRecordTest() throws Exception {
         String metadataTypeUrl = Any.pack(newMetadataUuid).getTypeUrl();
         CorfuDynamicRecord expectedRecord = new CorfuDynamicRecord(valTypeUrl,
             dynamicValMessage, metadataTypeUrl, dynamicMetadataMessage);
-
         Assert.assertEquals(expectedRecord, addedRecord);
         Assert.assertEquals(2, browser.printTable(NAMESPACE, TABLE_NAME));
+
+        CorfuDynamicRecord expectedRecord2 = new CorfuDynamicRecord(valTypeUrl,
+                dynamicValMessage, null, null);
```

Comment:
if metadata is null, the typeUrl is an empty string instead of a null.. I suggest extracting the typeUrl like L883 so that the test passes values like the actual code.

Reply:
Sure, I will fix the expectedRecord the same way.